tests/lightning_tensor/test_measurements_class.py

```diff
@@ -145,7 +152,38 @@ def test_probs_many_wires(self, n_qubits, n_targets, tol):
         mp = qml.probs(wires=range(n_targets))
 
         tape = qml.tape.QuantumScript(ops, [mp])
-        res = dev.execute(tape)
         ref = dq.execute(tape)
 
-        assert np.allclose(res, ref, atol=tol, rtol=0)
+        if method == "tn":
+            with pytest.raises(qml.DeviceError):
+                res = dev.execute(tape)
+        else:
+            res = dev.execute(tape)
+            assert np.allclose(res, ref, atol=tol, rtol=0)
+
+    @pytest.mark.parametrize("method", ["mps", "tn"])
+    @pytest.mark.parametrize("n_qubits", range(4, 14, 2))
+    @pytest.mark.parametrize("n_targets", list(range(1, 4)) + list(range(4, 14, 2)))
+    def test_state_many_wires(self, method, n_qubits, n_targets, tol):
+        """Test probs measuring many wires of a random quantum state."""
+        if n_targets >= n_qubits:
+            pytest.skip("Number of targets cannot exceed the number of wires.")
+
+        dev = qml.device(device_name, wires=n_qubits, method=method)
+        dq = qml.device("default.qubit", wires=n_qubits)
```

Comment:
It is preferable to avoid directly calling `default.qubit` in Lightning tests.
This creates a certain level of unpredictability, regarding changes in PennyLane. 
It is OK to calculate locally with `default.qubit` and add the results here, though.
If you NEED to call a device maybe `lightning.qubit`?

Reply:
Done!